packages/notifi-react-card/lib/components/intercom/NotifiIntercomFTUNotificationTargetSection.tsx

```diff
@@ -66,41 +64,6 @@ export const NotifiIntercomFTUNotificationTargetSection: React.FC<
           </div>
         </div>
       ) : null}
-      {data.contactInfo.sms.active ? (
-        <NotifiSmsInput
-          disabled={false}
-          classNames={classNames?.NotifiSmsInput}
-          allowedCountryCodes={allowedCountryCodes}
-          copy={{ label: inputLabels?.sms }}
-          intercomSmsInputContainerStyle={'NotifiIntercomSmsInput__container'}
```

Comment:
Can we please also remove related smsInput classnames in the `default.css`?

Reply:
good catch and ticket to remove sms in config tool has been created